components/ws-daemon/seccomp-profile-installer/main.go

```diff
@@ -42,7 +42,7 @@ func main() {
 			Action: specs.ActAllow,
 		},
 
-		// slirp4netns requires setns, as do we for debugging
+		// Running docker on a workspace requires setns
```

Comment:
docker is still using slirp4netns.

Reply:
We disabled the docker slirp4netns use when we introduced slirp4netns for the workspace as a whole. We have however not yet removed that code from docker-up. We should do that in a follow-up PR.